base/server/python/pki/server/deployment/pkiconfig.py

```diff
@@ -40,7 +40,6 @@
 PKI_DEPLOYMENT_DEFAULT_USER = "pkiuser"
 
 PKI_SUBSYSTEMS = ["CA", "KRA", "OCSP", "RA", "TKS", "TPS"]
-PKI_SIGNED_AUDIT_SUBSYSTEMS = ["CA", "KRA", "OCSP", "TKS", "TPS"]
 PKI_TOMCAT_SUBSYSTEMS = ["CA", "KRA", "OCSP", "TKS", "TPS"]
```

Comment:
PKI_TOMCAT_SUBSYSTEMS is no longer necessary either since everything is running on Tomcat. In the past TPS ran on Apache HTTPD.